tests/installation/partitioning_filesystem.pm

```diff
@@ -23,16 +24,31 @@ sub run {
     assert_and_click 'edit-proposal-settings';
 
     if (get_var('PARTITIONING_WARNINGS')) {
-        assert_screen 'proposal-will-overwrite-manual-changes';
-        send_key 'alt-y';
+        if (is_storage_ng && !check_screen 'proposal-will-overwrite-manual-changes', 0) {
+            record_soft_failure 'bsc#1055756';
+        }
+        else {
+            assert_screen 'proposal-will-overwrite-manual-changes';
+            send_key 'alt-y';
+        }
+    }
+    if (is_storage_ng) {
+        assert_screen 'partition-scheme';
+        send_key $cmd{next};
     }
     # select the combo box
     assert_and_click 'default-root-filesystem';
 
     # select filesystem
     assert_and_click "filesystem-$fs";
     assert_screen "$fs-selected";
-    assert_and_click 'ok-button';
+
+    if (is_storage_ng) {
+        send_key 'alt-n';
+    }
+    else {
+        assert_and_click 'ok-button';
+    }
```

Comment:
does the ok button also have a shortcut? Then I would rather use the same approach, i.e. either clicking or shortcut in both flows

Reply:
This was there before, so I left this code as it was before

Reply:
I believe myself that it's weak argument, so I've changed it =)